### ISSUE TITLE
Document SFC CSS v-bind types behavior

### ISSUE THREAD

**Issue by segevfiner** [vuejs/docs#1842]

**Comment by skirtles-code** [comment]
Agreed. I've seen people ask about this several times on Vue Land. Typically people expect to be able to do something like this:

```css
font-size: v-bind(size) + 'px';    
```

That won't work.

Wrapping the value in backticks does work, though I'm unclear whether this is official supported or just an accident of the implementation:

```css
font-size: v-bind(`${size}px`)    
```

[Playground](https://sfc.vuejs.org/#eNpNkMFugzAMhl/Fiia1lQZsV0Qr7R12zKEFzJaKJFZiunaId58NOzSn/H9+f449mw+i8jahqU2Tu+SIISNPdLLBeYqJYYaEAywwpOhhJ9GdDTZ0MWSJul+Eowb2728HG5pqY0i1CEZP44VRFEDjAk0M/CA8WhMm32KyBm6Fjz2OYinLmi2r7UHOJ955NSp1muqJKDLzY0TIXSTsxSGYNTvEwIXCaoG3LvT788useqH7+aBUG5b1q1p+Mq9mG7TwFyqvOQZZxQqy/w/ZmnpDqycLUG3NNzPluqry0OkCr7mM6auSW5mmwM5jidkXbYo/GZOArVGEdF7M8gcVCoER)

Based on what's currently documented, the 'correct' way to do this would be:

```css
font-size: v-bind('size + "px"')
```

[Playground](https://sfc.vuejs.org/#__DEV__eNp1kMFOwzAMhl/F8qVDrA1cq24S78Axl611odOSWIk7Nqq+O07LYQItp/jP7y/2P+Ebc3UZCWtsUhsHFkgkI++tHxyHKDBBpB5m6GNwUKi1sN76Nvik1uGbYJcNm9eXJ+sbszK0Wwshx+eDkFYAzeB5FJAb086iH92RokW4lC50dFYpsyyu3vw96HmnqyyCyUpj7ohaJrmdCVIbmDpVGKbs7YOXMsNqhR8H322KZcxnsMhXi4XOOS+j5vY9bnFdtHQHrk4peI1iAdnfh2SxXtFZ0wBybfFThFNtTOrbHOApVSF+GL1VcfQyOKooufIYw1eiqGCL2zuGUfFCsYzkO4o5isfMP9Z/3IzVjWacfwAeQp+d)

Or even ``font-size: v-bind('`${size}px`')``.

But this is not obvious.
